scripts/postinstall.js

```diff
@@ -2,13 +2,20 @@ const fs = require('fs');
 const crypto = require('crypto');
 const fname = 'secrets.env';
 
+// need to provide the api key
+// to both the bpanel server
+// and the wallet server
+const randomValue = crypto.randomBytes(40).toString('hex');
+
 // Init secrets.env
 if (!fs.existsSync(fname)) {
   fs.writeFileSync(
     fname,
-    'BCOIN_API_KEY=' + crypto.randomBytes(40).toString('hex') + '\n',
+    `BCOIN_API_KEY=${randomValue}
+    BCOIN_NODE_API_KEY=${randomValue}`,
```

Comment:
I think you'll still want the new line here so that there's an empty line at the end of the file.

Reply:
do we really want both keys to have the same string?

Reply:
Two different clients need the same api key but they use different env variables to get the key

Reply:
ahh, gotcha

Reply:
will add a comment to make that clear 👍